wiki/Help_centre/Account/en.md

```diff
@@ -206,6 +206,21 @@ We are considering extra 'score reset' purchases from the osu!store for the trul
 
 This doesn't mean you can make a new account to do the same thing, either! Having multiple accounts is against the [rules](/wiki/Rules#community-rules), and we will disable any such accounts that are brought to our attention.
 
+### I no longer want to keep playing, but I don't want my account to be deleted! 
```

Comment:
this section should be below the one about account deletion

Reply:
I felt that it was above the deletion because it *is* the stage of an account before the deletion of the account, and therefore should be read before the deletion option.

Reply:
well it doesn't flow as well when you skim the article

since you have "can i delete my account?", then "I no longer want to keep playing, but I don't want my account to be deleted!" follows naturally

if the title was "can i lock my account?" then i'd get your point, but i do think this is a good title as is